contracts/andromeda_gumball/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "andromeda_gumball"
+version = "0.1.0"
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[features]
+# for more explicit tests, cargo test --features=backtraces
+backtraces = ["cosmwasm-std/backtraces"]
+# use library feature to disable all instantiate/execute/query exports
+library = []
+
+[dependencies]
+cosmwasm-std = { version = "0.16.2" }
```

Comment:
Now that https://github.com/andromedaprotocol/andromeda-contracts/pull/152 has been merged, lets have all new contracts use the `0.16.7` version.